visualization/.prettierrc.json

```diff
@@ -0,0 +1,10 @@
+{
+    "printWidth": 140,
+    "tabWidth": 4,
```

Comment:
Did you check that the tabWith will be 4 spaces and not 8 (although declared with 4) like on the closed PR #339?

Reply:
Checked on local machine and it did 4 spaces tabs. As this PR doesn't apply the prettier, we'll check that again, when setting sup the actual changing PR.